webmails/rainloop/Dockerfile

```diff
@@ -1,4 +1,10 @@
-FROM php:7.2-apache
+ARG PHP_DISTRO=php:7.2-apache
+
+FROM $PHP_DISTRO
+# FIXME: hardcoded since : --from won't work with variables
+# and with multistage build --from does not seem to copy the binary
+COPY --from=balenalib/rpi-alpine:3.8 /usr/bin/qemu-arm-static /usr/bin/qemu-arm-static
```

Comment:
I'm aware of the limitations you expressed:
>the qemu file is needed when you build for arm from a non arm plateform. Most of my changes are just permitting a variable to override the image to use so the build script can use an arm image. The one I used in the build script already has the qemu except for the php image and I could not find any other way to do that.

However, this will affect non-arm build as well and therefore cannot be merged as-is. You'll have to find another solution. Perhaps a custom php-arm image containing the binary?

Reply:
@muhlemmer is it an issue to have the binay in non-arm machines as it won't interact with the machine when it's not running arm binaries ? It's not that big if we consider the size of the apache-php stack, and I don't think having an extra binary in that kind of setup changes much.

Reply:
@muhlemmer what if I add it for the build and remove it at the end, would that be ok ?